src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -161,6 +161,37 @@ describe('<DateSelector />', () => {
     });
   });
 
+  describe.only('timezones adjustments', () => {
+    const timeZoneOffsets = [720, 540, 60, 0, -60, -540, -720];
+    const dates = ['2017-12-01', '2019-01-01', '2019-08-22'];
+
+    const getTimezoneOffset = Date.prototype.getTimezoneOffset;
+
+    dates.forEach((date) => {
+      timeZoneOffsets.forEach((zone) => {
+        it(`sets the correct values for ${date} with timezone offset ${zone}`, () => {
```

Comment:
It feels a bit weird to have a test in nested `forEach`.

I would instead run one test for a given date and given offset. If it works for one offset, it should work for all offsets no?

Your `getTimezoneOffset` mock does not seem to be properly set.

If you want a mock for multiple tests you could do something like this I think?

```
beforeEach(() => {
    getTimezoneOffset = jest.spyOn(Date.prototype, 'getTimezoneOffset');
  });

  afterEach(() => {
    getTimezoneOffset.mockReset();
  });

In your test: 

getTimezoneOffset.mockImplementation(() => mockedOffset);

```